include/deal.II/dofs/dof_accessor.h

```diff
@@ -1828,7 +1828,7 @@ class DoFCellAccessor : public DoFAccessor<DoFHandlerType::dimension,
    * cells, if all degrees of freedom of the FiniteElement are located in
    * vertices. This functionality will vanish in a future release.
    */
-  void
+  DEAL_II_DEPRECATED void
   get_dof_indices(std::vector<types::global_dof_index> &dof_indices) const;
```

Comment:
Wait, I don't think this is right. The function itself is not deprecated at all and in fact used all over the place. It's just the one particular use mentioned in the comment that is deprecated!